app/_hub/kong-inc/rate-limiting/2.1-x.md

```diff
@@ -0,0 +1,233 @@
+</div>
+
+## Implementation considerations
+
+The plugin supports 3 policies, which each have their specific pros and cons.
+
+| policy    | pros                                                        | cons                                                                                                                                |
+| --------- | ----------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `cluster` | accurate, no extra components to support                    | relatively the biggest performance impact, each request forces a read and a write on the underlying datastore.                      |
+| `redis`   | accurate, lesser performance impact than a `cluster` policy | extra redis installation required, bigger performance impact than a `local` policy                                                  |
+| `local`   | minimal performance impact                                  | less accurate, and unless a consistent-hashing load balancer is used in front of Kong, it diverges when scaling the number of nodes |
+
+There are 2 use cases that are most common:
+
+1. _every transaction counts_. These are for example transactions with financial
+   consequences. Here the highest level of accuracy is required.
+2. _backend protection_. This is where accuracy is not as relevant, but it is
+   merely used to protect backend services from overload. Either by specific
+   users, or to protect against an attack in general.
+
+**NOTE**:
+
+<div class="alert alert-warning">
+  <strong>Enterprise-Only</strong> The Kong Community Edition of this Rate Limiting plugin does not
+include <a href="https://redis.io/topics/sentinel">Redis Sentinel</a> support.
+<a href="https://www.konghq.com/enterprise/">Kong Enterprise Subscription</a> customers have the option
+of using Redis Sentinel with Kong Rate Limiting to deliver highly available primary-replica deployments.
+</div>
+
+### Every transaction counts
+
+In this scenario, the `local` policy is not an option. So here the decision is between
+the extra performance of the `redis` policy against its extra support effort. Based on that balance,
+the choice should either be `cluster` or `redis`.
+
+The recommendation is to start with the `cluster` policy, with the option to move over to `redis`
+if performance reduces drastically. Keep in mind existing usage metrics cannot
+be ported from the datastore to redis. Generally with shortlived metrics (per second or per minute)
+this is not an issue, but with longer lived ones (months) it might be, so you might want to plan
+your switch more carefully.
+
+### Backend protection
+
+As accuracy is of lesser importance, the `local` policy can be used. It might require some experimenting
```

Comment:
This sentence feels like it's missing something. It just jumps into the middle with "As accuracy" ... accuracy of what / why is it less important?

Reply:
changed to `If accuracy...`, which is consistent with what the sentence tries to convey.